docroot/sites/all/modules/custom/ppgetstat/ppgetstat.module

```diff
@@ -68,51 +74,113 @@ function ppgetstat_menu() {
 }
 
 /**
- * Get drupal.org user ID using nickname and http://dgo.to.
+ * Get/set scanning period.
  *
- * @param string $nick
- *   Nickname.
+ * @param int $months
+ *   Number of months.
  *
  * @return int
- *   Drupal.org user id.
+ *   Timestamp.
  */
-function ppgetstat_get_id_by_nick($nick) {
-  if ($id_cached = cache_get('ppgetstat_nick_' . $nick)) {
-    return $id_cached->data;
+function ppgetstat_stats_period($months = NULL) {
```

Comment:
Getter/setter for scanning period. Needed to be able to use month instead of timestamps.